bin/configure-scripts/push_to_origin.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+require "#{__dir__}/utils"
+
+has_origin_remote = `git remote | grep origin`.strip.length > 0
```

Comment:
@jagthedrummer

I changed `origin` to `bullet-train` remote at the beginning as the script suggested, and near the end of the script I ran into this error:

```
Would you like to commit the changes to your project? [Y/n]
n

Make sure you save your changes with Git.
This repo doesn't appear to have an `origin` remote configured. Skipping pushing to `origin`.
/home/gazayas/work/bt/test_apps/bin_configure_beta_test/bin/configure-scripts/check_postgres.rb:5:in ``': No such file or directory - postgres (Errno::ENOENT)
	from /home/gazayas/work/bt/test_apps/bin_configure_beta_test/bin/configure-scripts/check_postgres.rb:5:in `<top (required)>'
	from <internal:/home/gazayas/.rbenv/versions/3.3.1/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
	from <internal:/home/gazayas/.rbenv/versions/3.3.1/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
	from ./bin/configure-beta:32:in `<main>'
> 
```

I see that you've added this line in another file, so maybe we should add it here too?
```ruby
has_bt_remote = `git remote | grep bullet-train`.strip.length > 0
```

Reply:
Thanks for giving it a shot @gazayas! I don't think this error actually had anything to do with the missing `origin`. It's related to some refactoring I did in trying to detect postgres.

It's complaining about this line:

https://github.com/bullet-train-co/bullet_train/blob/904c7bc0c087082be9bd0f3296665f5ebd858529/bin/configure-scripts/check_postgres.rb#L5

When I'm just trying to run the `postgres --version` command to detect if it's installed and what version. I've now wrapped that in a `begin`/`rescue` block so that it won't error out if the `postgres` command isn't able to be found. And I added a fallback to check for `psql` if `postgres` isn't found. (I also added some "section headers" to make it more clear which step of the config process we're on at any given time.) Can you give it another shot?

Also, I'm curious what platform are you running on? I had hoped we could get away from having to maintain platform and package-manager specific ways of detecting postgres (and redis & icu4c), but maybe that's wishful thinking...